apps/astarte_pairing/lib/astarte_pairing/cfssl_credentials.ex

```diff
@@ -42,6 +42,12 @@ defmodule Astarte.Pairing.CFSSLCredentials do
         cn = CertUtils.common_name!(cert)
 
         if cn == device_common_name do
+          Logger.info(
+            "Certificate issued successfully. ",
+            realm: realm,
+            hw_id: encoded_device_id
+          )
```

Comment:
on the other hand, on this event we need to also log the cn


```suggestion
          Logger.info(
            "Certificate issued successfully. ",
            realm: realm,
            hw_id: encoded_device_id,
            common_name: cn
          )
```

we should also the tag to the logger's metadat in the config file